install/kubernetes/cilium/values.yaml

```diff
@@ -106,13 +106,13 @@ affinity:
                 - linux
   podAntiAffinity:
     requiredDuringSchedulingIgnoredDuringExecution:
-    - labelSelector:
```

Comment:
This and other indentation changes to `value.yaml` seem unnecessary. If you would like to cleanup indentation, please put that into a separate commit, or better - another PR :)

Reply:
Sorry, these are caused by my own formatting tools